code/modules/jobs/job_types/ai.dm

```diff
@@ -71,3 +71,5 @@
 /datum/job/ai/config_check()
 	return CONFIG_GET(flag/allow_ai)
 
+/datum/job/ai/radio_help_message(mob/M)
+	to_chat(M, "<b>Prefix your message with :b to speak with cyborgs and other AIs.</b>")
```

Comment:
Undocumented change--put it in the changelog.

Reply:
I did tho? `fix: Fixed the AI radio help message giving false info about the generic department key.`

Reply:
Mistook that for something else, my bad.

Reply:
No worries. Might have been better for me to describe how the fix is an override.